spring-boot-modules/spring-boot-validations/src/test/java/com/baeldung/controller/ValidationControllerUnitTest.java

```diff
@@ -0,0 +1,138 @@
+package com.baeldung.controller;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.web.servlet.MockMvc;
+
+import com.baeldung.service.ValidationService;
+
+@ExtendWith(SpringExtension.class)
+@WebMvcTest(controllers = ValidationController.class)
+class ValidationControllerUnitTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @TestConfiguration
+    static class EmployeeServiceImplTestContextConfiguration {
+        @Bean
+        public ValidationService validationService() {
+            return new ValidationService() {
+            };
+        }
+    }
+
+    @Autowired
+    ValidationService service;
+
+    @Test
+    void whenNullInputForBooleanField_thenHttpBadRequestAsHttpResponse() throws Exception {
+
```

Comment:
Most of the methods here have a blank line at the beginning or end or both (after opening curly brace and before closing curly brace). As per guidelines, let's remove those.

Reply:
Updated as per review comment